conf/application.conf

```diff
@@ -117,6 +117,11 @@ feature-switch {
   version-1 {
     enabled = true
   }
+
+  release5 {
+    enabled = true
+    released-in-production = false
+  }
```

Comment:
Can we remove this. "released-in-production" isn't really going to be a thing for a test-support API that's restricted to ET. Also for "enabled",  the default is true and so it is one fewer place to remember to remove the switch from when we get around to it.